tests/sqlparser_common.rs

```diff
@@ -666,6 +666,23 @@ fn parse_select_with_table_alias() {
     );
 }
 
+#[test]
+fn parse_consecutive_queries() {
```

Comment:
Not sure what the best approach here is, or if there's even a better solution apart from disallowing _any_ unquoted keyword as an alias